source/appModules/hxmail.py

```diff
@@ -0,0 +1,35 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import appModuleHandler
+import UIAHandler
+from NVDAObjects.UIA.wordDocument import WordDocument
+
+class MailWordDocumentTreeInterceptor(WordDocument.treeInterceptorClass):
+
+	def _get_isAlive(self):
+		return super(MailWordDocumentTreeInterceptor,self).isAlive and self.rootNVDAObject.shouldCreateTreeInterceptor

```

Comment:
I don't quite understand the `shouldCreateTreeInterceptor` check here. Does the document get reused between read-only and editable messages? Is this going to be negatively affected if the user forces browse mode in an editable message; i.e. will the TreeInterceptor not die when it's supposed to?